Purchases/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -131,7 +131,7 @@ internal protocol StoreProductType {
     /// - SeeAlso: ``pricePerMonth``.
     var price: Decimal { get }
 
-    /// The price of this product using ``priceFormatter``.
+    /// The price of this product formatted for locale.
```

Comment:
Why this change?

Reply:
Only SK1 uses `priceFormatter` for `localizedPriceString`. SK2 uses the `displayPrice` so the comment was wrong 🤷‍♂️  Should we be using `priceFormatter` for both SK1 and SK2?